core/client/fs/src/main/java/alluxio/client/file/cache/store/LocalPageStore.java

```diff
@@ -91,14 +110,26 @@ public void put(PageId pageId, byte[] page) throws ResourceExhaustedException, I
     }
   }
 
+  private void createFileMetadata(PageId pageId, FileInfo fileInfo) throws IOException {
+    Path metaDataPath = getFileMetaDataPath(pageId);
+    Properties metadata = new Properties();
+    metadata.setProperty("LastModificationTimeMs",
```

Comment:
“LastModificationTimeMs” should be defined in a constant string and then referenced here